tests/mentorship_relation/test_dao_cancel_relation.py

```diff
@@ -60,7 +60,7 @@ def test_dao_receiver_cancel_mentorship_request(self):
         db.session.commit()
 
         DAO = MentorshipRelationDAO()
-        result = DAO.cancel_relation(self.second_user.id, self.mentorship_relation.id)
+        result = DAO.cancel_relation(self.second_user.id, self.mentorship_relation.id, 'Some reason to cancel the relation.')
```

Comment:
@bhawesh96 these lines are a bit too long. Try to refactor the code a bit maybe and follow [PEP8](https://www.python.org/dev/peps/pep-0008/) ( <80 characters per line). We might add automatic linting tests and these will not pass.